### PR TITLE
Add Opik

### PR DESCRIPTION
**Pull request by anmorgan24**
## Requirements

 * [X] This is Software as a Service not self hosted
 * [X] It has a free tier not just a free trial
 * [X] Pricing information is clearly visible without signup or phone calls
 * [X] The submission mentions what is free
 * [X] The submission is not already present in the list
 * [X] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1093,6 +1093,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
   * [Clair](https://askclair.ai/) - Clinical AI Reference. Students have free access to the professional tool suite, which includes Open Search, Clinical Summary, Med Review, Drug Interactions, ICD-10 Codes, and Stewardship. Additionally, a free trial for the professional suite is available.
   * [Langtrace](https://langtrace.ai) - enables developers to trace, evaluate, manage prompts and datasets, and debug issues related to an LLM application’s performance. It creates open telemetry standard traces for any LLM which helps with observability and works with any observability client. Free plan offers 50K traces/month.
   * [Ultra AI](https://ultraai.app) - AI command center for your product. Features include multi-provider AI gateway, prompts manager, caching, logs, analytics, model fallbacks and user rate limiting. Free forever plan offers 10k+ requests per month and all the features except caching.
+  * [Opik](https://github.com/comet-ml/opik) - Evaluate, test, and ship LLM applications across your dev and production lifecycles.
```
Comment (ripienaar) [inline README.md:1096]: This doesn’t appear to be a SaaS. While there is one called Comet its pricing isn’t made clear so don’t think this fits the list.

Comment (anmorgan24) [reply README.md:1096]: Hi @ripienaar, thanks for the question! Comet is indeed the company behind Opik (the product name), which might get a bit confusing. You should be able to access pricing on this page: 

https://www.comet.com/site/pricing/

At the top, you should be able to toggle between "MLOps platform" (the original Comet ML product) and Opik (the new LLM evaluation product). 

If this isn't clear enough, let me know and I'll try to find something clearer! 🙂 

Screenshot 2024-09-27 at 7 38 15 AM

Comment (ripienaar) [reply README.md:1096]: In that case comet is the free offering and what you should add here.

**Comment by anmorgan24** [comment]
I just updated the name to Comet Opik!

File: README.md [hunk 2]
```diff
@@ -1093,6 +1093,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
   * [Clair](https://askclair.ai/) - Clinical AI Reference. Students have free access to the professional tool suite, which includes Open Search, Clinical Summary, Med Review, Drug Interactions, ICD-10 Codes, and Stewardship. Additionally, a free trial for the professional suite is available.
   * [Langtrace](https://langtrace.ai) - enables developers to trace, evaluate, manage prompts and datasets, and debug issues related to an LLM application’s performance. It creates open telemetry standard traces for any LLM which helps with observability and works with any observability client. Free plan offers 50K traces/month.
   * [Ultra AI](https://ultraai.app) - AI command center for your product. Features include multi-provider AI gateway, prompts manager, caching, logs, analytics, model fallbacks and user rate limiting. Free forever plan offers 10k+ requests per month and all the features except caching.
+  * [Comet Opik](https://github.com/comet-ml/opik) - Evaluate, test, and ship LLM applications across your dev and production lifecycles.
```
Comment (ripienaar) [inline README.md:1096]: obviously the URL needs to go to the thing also.  This is a list for SaaS, if you dont link to the SaaS and if that SaaS doesn't company with the requirements we cant put it on the list.

**Comment by anmorgan24** [comment]
Got it. I just switched the link from the github repo to the SaaS product page. Let me know if you need me to make any more changes

**Comment by ripienaar** [comment]
Thanks

**Pull request merged**